www/chromedriver/Portfile

```diff
@@ -3,12 +3,12 @@
 PortSystem          1.0
 
 name                chromedriver
-version             97.0.4692.71
+version             98.0.4758.80
 categories          www
 platforms           darwin
 maintainers         {@catap korins.ky:kirill} openmaintainer
 license             Apache-2
-supported_archs     x86_64
+supported_archs     arm64 x86_64
```

Comment:
I guess that `arm64` should be also restricted since version 98?

Reply:
No, Google has provided M1 code for all available versions. Please see https://sites.google.com/chromium.org/driver/downloads.